lib/generator.js

```diff
@@ -158,7 +158,7 @@ const wrapComment = (comment, opts) =>
   comment
     .trimLeft()
     .replace(/<[\w.]+(\[])?>/g, type => wrapType(type.slice(1, -1), opts))
-    .replace(/`{[\w.\s]+(\(\))?}`/, link => {
+    .replace(/`{[\w.#\s]+(\(\))?}`/g, link => {
```

Comment:
Is there a need for a `g` flag?

Reply:
Yes, previously it would only replace the first link in line.